### PR TITLE
chore: APP-264 separate created from signed events on post timeline

### PR DESCRIPTION
**Pull request by blushi**
## Description

https://regennetwork.atlassian.net/browse/APP-264

This also fixes `GetTxsEvent` to fetch more than 100 items (default limits).

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

https://deploy-preview-2443--regen-marketplace.netlify.app/post/regen:13toVhoyn9zGbPQXYhRtFe7MSG29KF7CM7tHgE8nYyMNrH9NmWXt7wx.rdf

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@erikalogie @clevinson see testing instructions

**Comment by erikalogie** [comment]
@blushi shouldn't the signer be the same as the creator? i.e. Marie test (admin) with a link to the profile?

Screenshot 2024-08-27 at 11 04 23 AM

**Comment by blushi** [comment]
> @blushi shouldn't the signer be the same as the creator? i.e. Marie test (admin) with a link to the profile?

it could be that the creator did not sign the post but it got signed afterwards by someone else. On which post is it? I guess it might be on a post I created and then signed from the CLI with another address to test the timeline UI when "attach signature" feature was not yet implemented. If you give me the post IRI, I can check if that's the case or an actual issue.

**Comment by erikalogie** [comment]
> > @blushi shouldn't the signer be the same as the creator? i.e. Marie test (admin) with a link to the profile?
> 
> it could be that the creator did not sign the post but it got signed afterwards by someone else. On which post is it? I guess it might be on a post I created and then signed from the CLI with another address to test the timeline UI when "attach signature" feature was not yet implemented. If you give me the post IRI, I can check if that's the case or an actual issue.

Ah ok, yes, I was just testing the IRI that you put in the description: https://deploy-preview-2443--regen-marketplace.netlify.app/post/regen:13toVhoyn9zGbPQXYhRtFe7MSG29KF7CM7tHgE8nYyMNrH9NmWXt7wx.rdf

But it does seem that the user is missing the admin designation on another post, i.e.: https://deploy-preview-2443--regen-marketplace.netlify.app/post/regen:13toVh3mCMRXNviQeb92PByKXKedmvvjVqSY4pLV88o9N3C3Md8wvdX.rdf, though the correct name and profile shows up on the signing event.

**Comment by blushi** [comment]
> > > @blushi shouldn't the signer be the same as the creator? i.e. Marie test (admin) with a link to the profile?
> > 
> > 
> > it could be that the creator did not sign the post but it got signed afterwards by someone else. On which post is it? I guess it might be on a post I created and then signed from the CLI with another address to test the timeline UI when "attach signature" feature was not yet implemented. If you give me the post IRI, I can check if that's the case or an actual issue.
> 
> Ah ok, yes, I was just testing the IRI that you put in the description: https://deploy-preview-2443--regen-marketplace.netlify.app/post/regen:13toVhoyn9zGbPQXYhRtFe7MSG29KF7CM7tHgE8nYyMNrH9NmWXt7wx.rdf
> 
> But it does seem that the user is missing the admin designation on another post, i.e.: https://deploy-preview-2443--regen-marketplace.netlify.app/post/regen:13toVh3mCMRXNviQeb92PByKXKedmvvjVqSY4pLV88o9N3C3Md8wvdX.rdf, though the correct name and profile shows up on the signing event.

Could you have another look now? @erikalogie

**Comment by erikalogie** [comment]
LGTM!

File: web-marketplace/src/pages/Dashboard/MyEcocredits/MyEcocredits.tsx [hunk 1]
```diff
@@ -231,7 +231,7 @@ export const MyEcocredits = (): JSX.Element => {
     setTxModalTitle,
     signAndBroadcast,
   });
-
+  console.log(basketPutOpen, credits);
```
[the remark this answers is not in the document]
Comment (blushi) [reply web-marketplace/src/pages/Dashboard/MyEcocredits/MyEcocredits.tsx:234]: oops good catch

**Review by r41ph — APPROVED** [review]

**Pull request merged**